chrome_agent/background.html

```diff
@@ -0,0 +1,12 @@
+<!doctype html>
```

Comment:
Is this file still needed now that `manifest.json` has been updated to contain a `background.scripts` entry?